tests/framework/scheduler.py

```diff
@@ -73,7 +203,18 @@ def do_pytest_addoption(parser):
             action="store",
             type=int,
             default=default,
-            help="Concurrency level (max number of worker processes to spawn)."
+            help="Concurrency level (max number of worker \
+processes to spawn).",
+        )
+
+        parser.addoption(
```

Comment:
Maybe we should add these API changes in the commit message, so that it is a bit clearer how to run tests now